sdkservice/sdkservice.go

```diff
@@ -24,6 +30,42 @@ func SdkServiceURL(version string) string {
 	return fmt.Sprintf(sdkServiceURLFmt, version)
 }
 
+type SDKServiceClients map[string]sdkutilities.Client
+
+func (clients SDKServiceClients) GetSDKServiceClient(chainName, version string) (sdkutilities.Client, *apierrors.Error) {
```

Comment:
do we really need to pass chainName? I think sdk-services is not specific to a particular chain

```suggestion
func (clients SDKServiceClients) GetSDKServiceClient(version string) (sdkutilities.Client, *apierrors.Error) {
```

Reply:
Used chainname here for errors so that I thought it will be easy for debugging.

Reply:
And it will also be helpful to have chainname as we are getting version from specific chain details.

Reply:
the chain_name will be automatically included in the logging context (along with all the parameters of the http request)